scripts/validation/rules/tutorial-metadata-schema.json

```
@@ -30,7 +30,8 @@
         "type": "string"
       },
       "difficulty": {
-        "type": "string"
+        "type": "string",
+        "enum": ["Easy", "Intermediate", "Advanced"]
```

```suggestion
        "enum": ["beginner", "intermediate", "advanced"]
```
I think the more common range is beginner - intermediate advanced. Shall we go for that instead? Or otherwise easy medium hard.

---

Sounds good to me, @karlsoderby what do you think? 👀

---

Once we make a decision, we need to change them actually in the same PR, otherwise the linter will complain :-)

---

Hey @marqdevx so the idea is to change `easy` > `beginner`? Sounds good to me, it is a better label.